static/flow/views/plot-view.js

```diff
@@ -0,0 +1,65 @@
+var createPlotCanvas = function (className, id, parent, mouseDownHandler, mouseMoveHandler, mouseUpHandler, useManyplot) {
```

Comment:
Breaking the presentation out is not only more elegant and maintainable, but was probably the only way to beat some of this into shape.  I would love to have the time to do this on ALL of the modules.  The original design seems to be to just jam everything into massive js files :(